src/frequenz/channels/_broadcast.py

```diff
@@ -127,6 +127,15 @@ def name(self) -> str:
         """
         return self._name
 
+    @property
+    def latest(self) -> T | None:
```

Comment:
Not sure it is a good idea to add this directly on the channel, unless you have a clear usecase in mind already?

Reply:
No, not in particular, it just entered in the bag of private stuff that should be safe to expose as read-only. But thinking about the case we were talking today about only needing the last value, I was sort of wondering why even a receiver is needed at all if we can access the `latest` value from the channel directly.

I guess your answer will be something like "because the channel is sort of a private interface and should be used through senders and receivers", not sure what's the right balance between pragmatism and purism :D

Reply:
> I was sort of wondering why even a receiver is needed at all if we can access the latest value from the channel directly.

yes, that's exactly what a `Peekable` does.  And the `_latest` variable was added to support the `Peekable` implementation, so I'd rather remove that as well.

And not all channels are accessible.  We have the `new_receiver` interface everywhere in the sdk's data pipeline, and even in the microgrid client.  Those classes own the channels and give out only receivers.  So it is also easier to have the same interface everywhere.

Reply:
And it is one less hurdle once we start trying to go to multi-process world.

Reply:
OK, I will remove it.

Reply:
Done